lib/util/miq-password.rb

```diff
@@ -156,6 +144,13 @@ def self.ez_load(filename)
     File.exist?(filename) && EzCrypto::Key.load(filename)
   end
 
+  def self.ez_load_v0(filename)
```

Comment:
Right now, fix_auth does not support using a v0 key, I'm making these changes so we can update fix_auth to start using that key